cmd/cluster/context.go

```diff
@@ -726,13 +734,26 @@ func printServiceLogs(serviceLogs []sl.ServiceLogShort, verbose bool, sinceDays
 	if verbose {
 		marshalledSLs, err := json.MarshalIndent(serviceLogs, "", "  ")
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Couldn't prepare service logs for printing: %v", err)
+			_, _ = fmt.Fprintf(os.Stderr, "Couldn't prepare service logs for printing: %v", err)
 		}
-		dump.Pretty(os.Stdout, marshalledSLs)
+		_ = dump.Pretty(os.Stdout, marshalledSLs)
 	} else {
 		// Non verbose only prints the summaries
 		for i, errorServiceLog := range serviceLogs {
-			fmt.Printf("%d. %s (%s)\n", i, errorServiceLog.Summary, errorServiceLog.CreatedAt.Format(time.RFC3339))
+			var serviceLogSummary string
+			if errorServiceLog.InternalOnly {
+				internalServiceLogLines := strings.Split(errorServiceLog.Description, "\n")
+				if len(internalServiceLogLines) > 0 {
+					// if the description is "", Split returns []
+					serviceLogSummary = fmt.Sprintf("INT %s", internalServiceLogLines[0])
+				} else {
+					serviceLogSummary = errorServiceLog.Summary
```

Comment:
**Question:** Would it make sense to also format internal service logs without description in a way that we know it's an internal SL? Also, do you know if service logs without a description exist?

Reply:
I suppose you could have an internal service log without any meaningful message but osdctl will make you work for it. We could change the internal template to have the "INT" instead of printing it out here, but then the older ones wouldn't get formatted. Also, for the short form presented by cluster context, I like the indicator that it's an internal service log to be short. Other formats might prefer longer formats if they have more space. 

```
❯ osdctl servicelog post $CLUSTER_ID --internal
ERRO[0000] The one of the template files is using '${MESSAGE}' parameter, but '--param' flag is not set for this one. Use '-p MESSAGE="FOOBAR"' to fix this.
FATA[0000] Please define this missing parameter properly.
```